### PR TITLE
Update framework.rst

### PR DESCRIPTION
**Pull request by apodgorbunschih**
Adding the default_path option to translator configuration

### CODE REVIEW

File: reference/configuration/framework.rst [hunk 1]
```diff
@@ -1560,6 +1561,16 @@ paths
 This option allows to define an array of paths where the component will look
 for translation files.
 
+.. _reference-translator-default_path:
+
+default_path
+.....
+
+**type**: ``string`` **default**: ``[]``
```
Comment (xabbuh) [inline reference/configuration/framework.rst]: the default value looks wrong to me (this should be `%kernel.project_dir%/translations`)

Comment (apodgorbunschih) [reply reference/configuration/framework.rst]: Thanks, you are right

File: reference/configuration/framework.rst [hunk 2]
```diff
@@ -1560,6 +1561,16 @@ paths
 This option allows to define an array of paths where the component will look
 for translation files.
 
+.. _reference-translator-default_path:
+
+default_path
+.....
```
Comment (javiereguiluz) [inline reference/configuration/framework.rst]: The format we use for the docs is more strict that Markdown, so for example, the line below the heading must have the exact same length. Please, add the missing dots like this:

```rst
default_path
............
```

File: reference/configuration/framework.rst [hunk 3]
```diff
@@ -1560,6 +1561,16 @@ paths
...
+
+**type**: ``string`` **default**: ``%kernel.project_dir%/translations``
+
+This option allows to define a path where the component will write
```
Comment (javiereguiluz) [inline reference/configuration/framework.rst]: I'm not sure about this description. Should we change it for this other description?

```rst
This option allows to define the path where the application translations files
are stored.
```

File: reference/configuration/framework.rst [hunk 4]
```diff
@@ -1560,6 +1561,16 @@ paths
 This option allows to define an array of paths where the component will look
 for translation files.
 
+.. _reference-translator-default_path:
+
+default_path
+.....
+
+**type**: ``string`` **default**: ``%kernel.project_dir%/translations``
```
Comment (javiereguiluz) [inline reference/configuration/framework.rst]: When documenting a new feature, we put a warning about the Symfony version where it was introduced. Could you please put this before this line? Thanks!

```rst
default_path
............

.. versionadded:: 3.4
    The ``default_path`` option was introduced in Symfony 3.4.

**type**: ``string`` **default**: ...
```

**Review by javiereguiluz — CHANGES_REQUESTED** [review]
@apodgorbunschih thanks for contributing the docs for this missing config option. We appreciate it a lot and we thank you your first Symfony Docs contribution.

I left some comments because the format we use to write the docs is a bit strict, but you did it great. Cheers!

File: reference/configuration/framework.rst (hunk 3 above)
Comment (HeahDude) [reply reference/configuration/framework.rst]: Actually it's two default things: it is where the command generates translation files and where domains take precedence on other registrations.

**Comment by javiereguiluz** [comment]
@apodgorbunschih will you have some time to make the suggested changes and fix the merge conflicts? Please note: because of the way Symfony does things, those conflicts can't be solved with the `Resolve conflicts` button of GitHub. They must be solved in your local computer. If this is too much, you can create a new pull request or ask us to take over this. Thanks!

**Comment by apodgorbunschih** [comment]
@javiereguiluz thanks for feedback, sorry for long delay, i will try to fix the conflicts today or tomorrow.

**Comment by javiereguiluz** [comment]
@apodgorbunschih thanks a lot for your contribution and for making it ready for merge. Congrats on your first Symfony Docs contribution!

**Pull request closed without merging**